app/src/main/res/layout/ar_layout.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/constraintLayout"
+    android:id="@+id/constraintLayout2"
```

Comment:
This was just a tryout. Doesn't seem like it matters, just wanted it to have a unique name.